gradle/libs.versions.toml

```diff
@@ -17,7 +17,7 @@ jupiter-jupiter-params = { module = 'org.junit.jupiter:junit-jupiter-params' }
 jupiter-api = { module = 'org.junit.jupiter:junit-jupiter-api' }
 junit-jupiter-engine = { module = 'org.junit.jupiter:junit-jupiter-engine' }
 
-micronaut-cache-core = { module = 'io.micronaut.cache:micronaut-cache-core', version = "3.4.1" }
+micronaut-cache-core = { module = 'io.micronaut.cache:micronaut-cache-core', version = "3.5.0" }
```

Comment:
@timyates can we remove the version?

Reply:
All this is removed in the Serde PR (incoming)